### PR TITLE
Adding GL_ARM_texture_unnormalized_coordinates

### PR DESCRIPTION
**Pull request by janharaldfredriksen-arm**

### CODE REVIEW

File: api/GL/glcorearb.h [hunk 1]
```diff
@@ -3838,6 +3838,12 @@ GLAPI void APIENTRY glTexBufferARB (GLenum target, GLenum internalformat, GLuint
 
 #ifndef GL_ARB_viewport_array
 #define GL_ARB_viewport_array 1
+typedef void (APIENTRYP PFNGLDEPTHRANGEARRAYDVNVPROC) (GLuint first, GLsizei count, const GLdouble *v);
```
Comment (janharaldfredriksen-arm) [inline api/GL/glcorearb.h]: These are unrelated to my changes. It looks like there's a recent commit that didn't regenerate the headers.

**Comment by pdaniell-nv** [comment]
@janharaldfredriksen-arm do you want any group time to potentially make this multi-vendor, or would you rather just get it in the registry asap?

**Comment by janharaldfredriksen-arm** [comment]
@pdaniell-nv I'd rather just get it in asap. (Or at least, we'll need this to ship as a vendor for now, if there's interest I'm fine with a multi-vendor later.)

**Comment by pdaniell-nv** [comment]
@oddhack this is approved to merge. Thanks.

File: xml/gl.xml [hunk 2]
```diff
@@ -44079,6 +44080,11 @@ typedef unsigned int GLhandleARB;
             </require>
         </extension>
         <extension name="GL_ARM_shader_framebuffer_fetch_depth_stencil" supported="gles2"/>
+        <extension name="GL_ARM_texture_unnormalized_coordinates" supported="gles2">
+            <require>
+                <enum name="GL_TEXTURE_UNNORMALIZED_COORDINATES_ARM"/>
```
Comment (Perksey) [inline xml/gl.xml]:
```suggestion
                <enum name="GL_TEXTURE_UNNORMALIZED_COORDINATES_ARM" group="SamplerParameterF,SamplerParameterI,GetTextureParameter,TextureParameterName" />
```

Comment (janharaldfredriksen-arm) [reply xml/gl.xml]: @Perksey  - Thanks. I'm not very familiar with these group parameters, but it looks correct as far as I can tell. Happy to take the suggestion if you think it's complete?

Comment (oddhack) [reply xml/gl.xml]: I will leave this open for a little while to sort this out. We are basically leaving the group stuff to outside developers like @Perksey so long as it doesn't affect the headers.

Comment (SunSerega) [reply xml/gl.xml]: Doesn't it make more sense to add `group` attribute on line 10342, instead of 44085?

Comment (Perksey) [reply xml/gl.xml]: Ah whoops, yep that's wrong. Thanks for picking up on that @SunSerega

File: xml/gl.xml [hunk 3]
```diff
@@ -10339,7 +10339,8 @@ typedef unsigned int GLhandleARB;
         <enum value="0x8F67" name="GL_MAX_SHADER_PIXEL_LOCAL_STORAGE_SIZE_EXT"/>
             <unused start="0x8F68" vendor="ARM"/>
         <enum value="0x8F69" name="GL_TEXTURE_ASTC_DECODE_PRECISION_EXT"/>
-            <unused start="0x8F6A" end="0x8F6F" vendor="ARM"/>
+        <enum value="0x8F6A" name="GL_TEXTURE_UNNORMALIZED_COORDINATES_ARM"/>
```
Comment (Perksey) [inline xml/gl.xml]:
```suggestion
        <enum value="0x8F6A" name="GL_TEXTURE_UNNORMALIZED_COORDINATES_ARM" group="SamplerParameterF,SamplerParameterI,GetTextureParameter,TextureParameterName"/>
```

File: xml/gl.xml (hunk 2 above)
Comment (janharaldfredriksen-arm) [reply xml/gl.xml]: Resolving this conversation - will take the other suggestion.

File: xml/gl.xml (hunk 3 above)
Comment (janharaldfredriksen-arm) [reply xml/gl.xml]: I couldn't get the Commit suggestion to work, but pushed the same change manually. Sanity-checked that it didn't change headers (other than generating new dates). Resolving.

File: xml/gl.xml [hunk 4]
```diff
@@ -10339,7 +10339,8 @@ typedef unsigned int GLhandleARB;
         <enum value="0x8F67" name="GL_MAX_SHADER_PIXEL_LOCAL_STORAGE_SIZE_EXT"/>
             <unused start="0x8F68" vendor="ARM"/>
         <enum value="0x8F69" name="GL_TEXTURE_ASTC_DECODE_PRECISION_EXT"/>
-            <unused start="0x8F6A" end="0x8F6F" vendor="ARM"/>
+        <enum value="0x8F6A" name="GL_TEXTURE_UNNORMALIZED_COORDINATES_ARM" group="SamplerParameterF,SamplerParameterI,GetTextureParameter,TextureParameterName"/>/>
```
Comment (Perksey) [inline xml/gl.xml]: Somehow you ended up ending the line twice as well, making the XML invalid. Don't know how that happened, probably my fault.
```suggestion
        <enum value="0x8F6A" name="GL_TEXTURE_UNNORMALIZED_COORDINATES_ARM" group="SamplerParameterF,SamplerParameterI,GetTextureParameter,TextureParameterName"/>
```

Comment (janharaldfredriksen-arm) [reply xml/gl.xml]: Probably me with fat fingers... Thanks for catching it.

Fixed (and squashed).

**Comment by janharaldfredriksen-arm** [comment]
@oddhack - it looks like this is stable now.
